cogs/fitwide.py

```diff
@@ -167,53 +168,98 @@ async def send_masstag_messages(self, ctx, prefix, target_ids):
-        # guild = self.bot.get_guild(config.guild_id)
-        guild = ctx.guild
-
-        BIT_names = [str(x) + "BIT" + ("+" if x == 4 else "") for x in range(5)]
-        BIT = [disnake.utils.get(guild.roles, name=role_name) for role_name in BIT_names]
+    @commands.slash_command(name="increment_roles", description=Messages.increment_roles_brief)
+    async def increment_roles(self, inter: disnake.ApplicationCommandInteraction):
+        await inter.send(Messages.increment_roles_start)
+        message = await inter.original_message()
+        guild = inter.guild
 
-        MIT_names = [str(x) + "MIT" + ("+" if x == 3 else "") for x in range(4)]
-        MIT = [disnake.utils.get(guild.roles, name=role_name) for role_name in MIT_names]
+        BIT_NAMES = ["0BIT", "1BIT", "2BIT", "3BIT+"]
+        BIT = [disnake.utils.get(guild.roles, name=role_name) for role_name in BIT_NAMES]
 
-        for member in BIT[3].members:
-            await member.add_roles(BIT[4])
-        for member in MIT[2].members:
-            await member.add_roles(MIT[3])
+        MIT_NAMES = ["0MIT", "1MIT", "2MIT+"]
+        MIT = [disnake.utils.get(guild.roles, name=role_name) for role_name in MIT_NAMES]
 
-        BIT_colors = [role.color for role in BIT]
+        # increment roles and create 0bit and 0mit
+        BIT_COLORS = [role.color for role in BIT]
         await BIT[3].delete()
-        await BIT[2].edit(name="3BIT+", color=BIT_colors[3])
-        await BIT[1].edit(name="2BIT", color=BIT_colors[2])
-        await BIT[0].edit(name="1BIT", color=BIT_colors[1])
-        bit0 = await guild.create_role(name="0BIT", color=BIT_colors[0])
+        await BIT[2].edit(name="3BIT+", color=BIT_COLORS[3])
+        await BIT[1].edit(name="2BIT", color=BIT_COLORS[2])
+        await BIT[0].edit(name="1BIT", color=BIT_COLORS[1])
+        bit0 = await guild.create_role(name="0BIT", color=BIT_COLORS[0])
         await bit0.edit(position=BIT[0].position - 1)
 
-        MIT_colors = [role.color for role in MIT]
+        MIT_COLORS = [role.color for role in MIT]
         await MIT[2].delete()
-        await MIT[1].edit(name="2MIT+", color=MIT_colors[2])
-        await MIT[0].edit(name="1MIT", color=MIT_colors[1])
-        mit0 = await guild.create_role(name="0MIT", color=MIT_colors[0])
+        await MIT[1].edit(name="2MIT+", color=MIT_COLORS[2])
+        await MIT[0].edit(name="1MIT", color=MIT_COLORS[1])
+        mit0 = await guild.create_role(name="0MIT", color=MIT_COLORS[0])
         await mit0.edit(position=MIT[0].position - 1)
 
-        general_names = [str(x) + "bit-general" for x in range(4)]
-        terminy_names = [str(x) + "bit-terminy" for x in range(1, 3)]
+        # status update
+        await message.edit(Messages.increment_roles_names)
+
+        # get channels by name
+        general_names = ["0bit-general", "1bit-general", "2bit-general", "3bit-general"]
+        terminy_names = ["1bit-terminy", "2bit-terminy", "3bit-terminy"]
```

Comment:
I think here it makes more sense to keep the original logic but use fstring.
```suggestion
        general_names = [f"{x}bit-general" for x in range(4)]
        terminy_names = [f"{x}bit-terminy" for x in range(1, 3)]
```